NewSpring/crete
# chore: Upgrade 1.5

## DESCRIPTION

### What does this PR do, or why is it needed?
This upgrades the API to core version 1.5.0.

Of note, this upgrade includes a fix for the auth bug we've been experiencing for a long time. We also are now using a core Bugsnag package for our API errors. 

### How do I test this PR?
To test you will have to run the API locally on this branch (`upgrade-1.5`) and the client on the `master` branch. The client should work just like it always has without any changes. Just test all the things.

You can use this doc as a guide: https://docs.google.com/document/d/1JoBErzORE2pMSfYS-79dw4O31RxKpMmE4vJPTNmU8uc/edit

Also, here is a list of all the 1.5.0 changes: https://docs.google.com/document/d/13_roIFoqiX2_gwQ6OvSnnwFT1cknxYWiDctHeggDbZI/edit#

**Note**: You'll have a couple of changes in your client directory after you've run the app. The files that get changed are `schema.graphql` and `fragmentTypes.json`. These changes will be picked up in the client side upgrade PR so you can safely remove the changes once you're done testing.

**Note 2**: Please check that videos work. I am having issues getting them to work correctly in my sim, but it's not just the upgrade that's causing the problem. I couldn't get them to work on `master` either ... but they are working in the app so ...

## TODO


## REVIEW


---

> The purpose of a PR Review is to _improve the quality of the software._
